### DISCUSSION TITLE
Error al despues de hacer el deploy de app-photoz-authz-uma

### DISCUSSION THREAD

**Discussion by iriartmaxim** [keycloak/keycloak#10208] [Q&A]
Que tal, estoy intentando desplegar app-photoz-authz-uma y al loguearme despues de hacer el deploy me da este error.
Alguien podría darme una mano? por favor jeje

![imagen](https://user-images.githubusercontent.com/97992995/153922935-6adb2ce5-6b59-4bc0-88e8-90b28575e2b5.png)

**Comment by stianst** [comment]
Please write questions in English
